### DISCUSSION TITLE
this thing stopped working what could be issue

### DISCUSSION THREAD

**Discussion by dirrickzm** [chr233/ASFEnhance#126] [提问求助 / Support]
**it was working fine from start of the sale now this happened i have updated to latest version but still**

![image](https://github.com/chr233/ASFEnhance/assets/62340423/18176879-ebec-4bee-9979-87a2d1b11a01)

**Comment by chr233** [comment]
🤔

**Reply by dirrickzm** [reply]
ok so when i try to farm steam sale trading cards now it gives these errors instead of farming

**Reply by dirrickzm** [reply]
these errors when i type !explorer in chat for card drops

**Comment by woctezuma** [comment]
![Picture](https://i.imgur.com/8dvazt6.png)

**Reply by dirrickzm** [reply]
yeh i removed that account

**Comment by chr233** [comment]
maybe due to your account is limited

**Reply by dirrickzm** [reply]
nah all of em are unres expect one which i removed well it worked after restarting internet
